client/ionic/src/components/TopNav.tsx

```diff
@@ -7,7 +7,7 @@ const TopNav: React.FC = () => {
       <IonToolbar>
         <IonItem lines="none" href="/menu">
           <IonImg
-            className="w-80 center h3"
+            className="w-80 h3 center pb3"
```

Comment:
```suggestion
            className="w-80 h3"
```

Let's keep design changes that are not related to the survey out of this PR.